docs/user/README.md

```diff
@@ -401,6 +432,23 @@ The wrap command can be used to wrap a given target with a pair of symbols
 - `"round wrap <TARGET>"`: wraps the target with parentheses
 - `"box wrap <TARGET>"`: wraps the target with square brackets
 
+### Slice
+
+The slice command is used to add multiple cursors to the editor. Each cursor is inserted at the same column on each row requested within the command.
+
+- `"pre <TARGET 1> slice <TARGET 2>"`
+- `"pre <TARGET 1> slice (past|until) <TARGET 2>"`
+
+The first command is equivalent to the second command when `"past"` is used. When `"until"` is used in the second command, cursors are inserted from the first target through to the line above the second target.
+
+### Reverse/Shuffle/Sort
+
+These commands are to be paired with the `"item"` scope and can be used to reorder items within a list.
```

Comment:
You can really pair them with anything you want.  I will sometimes do eg "sort line air slice bat", or select a bunch of things and say "sort this", etc